lecture_1/cli_args.js

```diff
@@ -1,26 +1,40 @@
-const fs = require("fs");
 const { stat } = require("fs/promises");
 
-const filePath = process.argv[2];
+const args = process.argv.slice(2);
+const pathFlagInd = args.findIndex(arg => arg === '--path');
 
-console.log(`Getting stats of ${filePath}`);
+if (pathFlagInd < 0) {
+  throw new Error('--path should be specified');
+}
 
-// fs.stat(filePath, (err, stats) => {
-//   if (err) {
-//     throw err;
-//   }
+const filePath = args[pathFlagInd + 1];
+const isEntityTypeRequired = args.findIndex(arg => arg === '--entity-type') > -1 ? true : false;
+
+console.log(`Getting stats of ${filePath} ${isEntityTypeRequired ? 'with' : 'without'} entity type`);
 
-//   console.log(stats);
-// });
 
 (async () => {
+
   try {
-    const metaData = await stat(filePath);
 
+    const metaData = await stat(filePath);
     console.log(metaData);
+
+    if (isEntityTypeRequired) {
```

Comment:
I'd rather use `switch case` construction and use immediate return to avoid nesting

```
if (!isEntityTypeRequired) {
  return
}

switch
...
```